templates/addCourse.phtml

```diff
@@ -48,14 +48,12 @@ $trainersCheckbox = \Portal\ViewHelpers\TrainerCheckboxViewHelper::displayTraine
 
         <div class="form-group">
             <label for="in_person">In Person?</label>
-            <input type="checkbox" id="in_person" value="1" class="create-courses" name="in_person">
-            <input id="in_person_spaces" name="in_person_spaces" class="create-courses form-control hidden" type="number" placeholder="How many spaces available?" min="1" max="999" data-required="true" />
+            <input type="radio" id="in_person" value="1" class="create-courses" name="radio">
```

Comment:
"radio" is not a good name for these two inputs, because that will become the key for the data that's submitted. Something that describes what you're selecting would be better.